models/servicenow__ticket_enhanced.sql

```diff
@@ -96,8 +99,8 @@ select
   assignee.department_value as assignee_department_value,
   assignee.sys_user_name as assignee_name,
   assignee.roles as assignee_roles,
-  task.task_closed_at,
-  {{ dbt.datediff("task.task_created_at", "task.task_closed_at", 'minute') }} as task_minutes_to_close,
+  task.task_closed_at, 
```

Comment:
Ah thanks for catching this-- it should be using opened_at, not created_at. After swapping, I can see that opened_at <= closed_at and the minutes are positive now.]

I also realized problem_minutes_to_resolve can be null because problem_resolved_at can be null, so I removed that calculated field